elizabeth/data/da/personal.json

```diff
@@ -2022,5 +2022,36 @@
 		"Test"
 	  ]
 	}
+  },
+  "personality": {
+	  "mbti": [
+		  "ISFJ",
+		  "ISTJ",
+		  "INFJ",
+		  "INTJ",
+		  "ISTP",
+		  "ISFP",
+		  "INFP",
+		  "INTP",
+		  "ESTP",
+		  "ESFP",
+		  "ENFP",
+		  "ENTP",
+		  "ESTJ",
+		  "ESFJ",
+		  "ENFJ",
+		  "ENTJ"
+	  ],
+	  "rheti": [
```

Comment:
It's duplicated in all 17 locales. Digits can be generated in method `personality()`

For example:

```python
if category == 'rheti':
    return randint(1, 10)
```
Please, fix it.